test/lib/open_days_test.rb

```diff
@@ -0,0 +1,72 @@
+require "test_helper"
+
+class ActivityNotifierTest < ActiveSupport::TestCase
```

Comment:
I think this should probably be:

```suggestion
class OpenDaysTest < ActiveSupport::TestCase
```